src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -525,6 +525,10 @@ public Response ondertekenInformatieObject(@PathParam("uuid") final UUID uuid,
                              policyService.readDocumentRechten(enkelvoudigInformatieobject,
                                                                zrcClientService.readZaak(zaakUUID)).getOndertekenen());
         enkelvoudigInformatieObjectOndertekenService.ondertekenEnkelvoudigInformatieObject(uuid);
+
+        // Hiervoor wordt door open zaak geen notificatie verstuurd. Dus zelf het ScreenEvent versturen!
+        eventingService.send(ENKELVOUDIG_INFORMATIEOBJECT.updated(uuid));
```

Comment:
Als je het document hebt, gebruik dan liever de updated method die een EnkelvoudigInformatieobject als parameter heeft. Die "weet" wat de juiste/gewenste sleutel is.

Reply:
Aangepast